Add blog post for April 1 community call (#178)
publiccodenet/blog: _posts/2021-04-12-notes-from-community-call-1-april-2021.md

---
title: "Notes from the Standard for Public Code community call - 1 April 2021"
date: 2021-04-12
author: Jan Ainali
type: blogpost
excerpt: Pruning the oldest issues
categories:
  - Community call
---

> # 1 April 2021 Standard for Public Code community call - internal notes

Reviewer: ```suggestion
# 1 April 2021 Standard for Public Code community call - notes
```

## Attendees

* [Jan Ainali](https://publiccode.net/who-we-are/team/jan-ainali.html)
* [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html)
* [Alba Roza](https://publiccode.net/who-we-are/team/alba-roza.html)

> * Jacob Green

Reviewer: ```suggestion
* [Jacob Green](https://twitter.com/jacoblyopen)
```

## Updates from the Foundation

- [Version 0.2.1 of the Standard for Public Code has been released](https://github.com/publiccodenet/standard/releases/tag/0.2.1)
- [Signalen has been replicated by 's-Hertogenbosch](https://commonground.nl/news/view/418b95d8-b3e3-4bd0-9ece-48984a472b93/gemeente-s-hertogenbosch-live-met-signalen) ([live site](https://meldingen.s-hertogenbosch.nl/incident/beschrijf))
- [Signalen had a community day](https://signalen.org/en/news/2021-03-29-notes-community-meetup/)

## Notes

> In this call we gave a thourough introduction to the Foundation of Public Code and how we work. Then we moved on to the [long backlog of open issues](https://github.com/publiccodenet/standard/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc) in the repository for Standard for Public Code and worked through them oldest to newest. This is the result of our pruning, which is also reflected in the GitHub repository:

Reviewer: ```suggestion
In this call we gave a thorough introduction to the Foundation of Public Code and how we work. Then we moved on to the [long backlog of open issues](https://github.com/publiccodenet/standard/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc) in the Standard for Public Code repository and worked through them from oldest to newest. This is the result of our pruning, which is also reflected in the GitHub repository:
```

> - [#38: Standardise terms for policy, civic code etc](https://github.com/publiccodenet/standard/issues/38) - closed

Reviewer: ```suggestion
- [#38: Standardize terms for policy, civic code etc](https://github.com/publiccodenet/standard/issues/38) - closed
```

Author: No, it's not a typo by me, this is actually the title of the issue.
